widgets/executions/src/tasksGraph/GraphNodes.tsx

```diff
@@ -1,29 +1,24 @@
-// @ts-nocheck File not migrated fully to TS
-import GraphNodePropType from './props/GraphNodePropType';
+import type { ElkNode } from 'elkjs';
+import type { Toolbox } from 'app/utils/StageAPI';
 import GraphNode from './GraphNode';
 import GraphEdges from './GraphEdges';
 
-/**
- * @property {Array} [graphNodes] - Array of Graph Nodes to render
- */
-
 const textVisualAdjustment = 5;
 
-const GraphNodes = ({ graphNodes, toolbox }) =>
-    graphNodes.map(graphNode => (
-        <g key={graphNode.id} transform={`translate(${graphNode.x}, ${graphNode.y + textVisualAdjustment})`}>
-            <GraphNode graphNode={graphNode} toolbox={toolbox} />
-            {!_.isEmpty(graphNode.children) && (
-                <>
-                    <GraphNodes graphNodes={graphNode.children} toolbox={toolbox} />
-                    <GraphEdges graphEdges={graphNode.edges} />
-                </>
-            )}
-        </g>
-    ));
-
-GraphNodes.propTypes = {
-    graphNodes: PropTypes.arrayOf(GraphNodePropType).isRequired
-};
+const GraphNodes = ({ graphNodes, toolbox }: { graphNodes: ElkNode[]; toolbox: Toolbox }) => (
+    <>
+        {graphNodes.map(graphNode => (
+            <g key={graphNode.id} transform={`translate(${graphNode.x}, ${graphNode.y! + textVisualAdjustment})`}>
+                <GraphNode graphNode={graphNode} toolbox={toolbox} />
+                {!_.isEmpty(graphNode.children) && (
```

Comment:
WDYT about using explicit lodash import here, as stated in [Ongoing Refactors Guideliness](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2148073473/Ongoing+Refactors+in+Cloudify+UI#Using-explicit-imports)? 🤔 

```suggestion
import type { ElkNode } from 'elkjs';
import type { Toolbox } from 'app/utils/StageAPI';
import { isEmpty } from 'lodash';
import GraphNode from './GraphNode';
import GraphEdges from './GraphEdges';

const textVisualAdjustment = 5;

const GraphNodes = ({ graphNodes, toolbox }: { graphNodes: ElkNode[]; toolbox: Toolbox }) => (
    <>
        {graphNodes.map(graphNode => (
            <g key={graphNode.id} transform={`translate(${graphNode.x}, ${graphNode.y! + textVisualAdjustment})`}>
                <GraphNode graphNode={graphNode} toolbox={toolbox} />
                {!isEmpty(graphNode.children) && (
```

Reply:
Done